### PR TITLE
Exposure Control component reference doc

### PR DESCRIPTION
**Pull request by chanmosq**
@netlify /docs/user-guide/components/reference/atom/exposure-control/

## Change summary

Author document for the Exposure Control component reference. 

### Submission Checklist:

* [x] **Descriptive active voice** - Do descriptive sentences have a clear *subject* and *action verb*?
* [x] **Answer the question at hand** - Does the documentation answer a *what*, *why*, *how*, or *where* type of question?
* [x] **Consistency** - Does the content consistently follow the [style guide](https://o3de.org/docs/contributing/to-docs/style-guide/)?
* [x] **Help the user** - Does the documentation show the user something *meaningful*?

### CODE REVIEW

**Review by sptramer — APPROVED** [review]

File: content/docs/user-guide/components/reference/atom/exposure-control.md [hunk 1]
```diff
@@ -5,16 +5,42 @@ description: 'Open 3D Engine (O3DE) Exposure Control component reference.'
...
+| **Eye Adaptation** | See [Eye Adaptation properties](#eye-adaptation-properties).  |   |   |
+
+
+### Eye Adaptation properties
+
+The following set of properties are active when **Control Type** is set to `Eye Adaptation`.
+
+| Property | Description | Value | Default |
+| - | - | - | - |
+| **Minimum Exposure** | The minimum exposure value at which automatic exposure can automatically adjust to. | Float: -16.0 - 16.0 | `-16.0` |
```
Comment (FiniteStateGit) [inline content/docs/user-guide/components/reference/atom/exposure-control.md:42]:
```suggestion
| **Minimum Exposure** | The minimum value that the exposure automatically adjusts to. | Float: -16.0 - 16.0 | `-16.0` |
```

File: content/docs/user-guide/components/reference/atom/exposure-control.md [hunk 2]
```diff
@@ -5,16 +5,42 @@ description: 'Open 3D Engine (O3DE) Exposure Control component reference.'
...
+| **Maximum Exposure** | The maximum exposure value at which automatic exposure can automatically adjust to. | Float: -16.0 - 16.0 | `16.0` |
```
Comment (FiniteStateGit) [inline content/docs/user-guide/components/reference/atom/exposure-control.md:43]:
```suggestion
| **Maximum Exposure** | The maximum value that the exposure automatically adjusts to. | Float: -16.0 - 16.0 | `16.0` |
```

File: content/docs/user-guide/components/reference/atom/exposure-control.md [hunk 3]
```diff
@@ -5,16 +5,42 @@ description: 'Open 3D Engine (O3DE) Exposure Control component reference.'
...
+| **Speed Up** |The speed at which automatic exposure adapts to brighter scenes. A higher value increases the speed. | Float: 0.01 - 10.0 | `3.0` |
+| **Speed Down** | The speed at which automatic exposure adapts to darker scenes. A higher value increases the speed. | Float: 0.01 - 10.0 |  `1.0` |
+| **Enable Heatmap** | Provides a histogram of the exposure values that the camera captures in the scene. It appears on top of the **Viewport**. You can move the camera to see the heatmap of other areas. Areas below the **Minimum Exposure** value are highlighted in blue, and areas above the **Maximum Exposure** value in red. The numbers on the heatmap represent stops. | None | None |
```
Comment (FiniteStateGit) [inline content/docs/user-guide/components/reference/atom/exposure-control.md:46]:
```suggestion
| **Enable Heatmap** | Displays a histogram of the exposure values that the camera captures in the scene. It appears on top of the **Viewport**. You can move the camera to see the heatmap of other areas. Areas below the **Minimum Exposure** value are highlighted in blue, and areas above the **Maximum Exposure** value in red. The numbers on the heatmap represent exposure value stops. | None | `Disabled` |
```

File: content/docs/user-guide/components/reference/atom/exposure-control.md [hunk 4]
```diff
@@ -5,16 +5,42 @@ description: 'Open 3D Engine (O3DE) Exposure Control component reference.'
 toc: true
 ---
 
-The **Exposure Control** component adjusts the amount of light the camera exposes in the scene, which controls how bright the scene appears. You can set the exposure manually or automatically using **Eye Adaptation** mode.
+The **Exposure Control** component adjusts the amount of light the camera exposes in the scene, which controls how bright the scene appears. You can set the exposure manually, or use *Eye Adaptation* mode to automatically adjust exposure based on the scene's brightness levels. Eye Adaptation mode adjusts the lighting in the scene so that the average brightness is at a middle gray lightness level. On top of the automatic exposure, you can further adjust the exposure by increasing or decreasing the **Manual Compensation** property, or exposure stops.
```
Comment (FiniteStateGit) [inline content/docs/user-guide/components/reference/atom/exposure-control.md:8]:
```suggestion
The **Exposure Control** component adjusts the amount of light from the scene that the camera receives, which controls how bright the scene appears. You can set the exposure manually, or use *Eye Adaptation* mode to automatically adjust camera exposure based on the scene's brightness. Eye Adaptation mode adjusts the scene to a neutral middle gray lightness. You can make further adjustments to the automatic exposure by increasing or decreasing the **Manual Compensation** property.
```

**Review by FiniteStateGit — CHANGES_REQUESTED** [review]

File: content/docs/user-guide/components/reference/atom/exposure-control.md [hunk 5]
```diff
@@ -5,16 +5,42 @@ description: 'Open 3D Engine (O3DE) Exposure Control component reference.'
...
-Add properties.
-{{< /todo >}}
+![Exposure Control interface](/images/user-guide/components/reference/atom/exposure-control/exposure-control-component-ui.png)
+
+### Base properties
+
+| Property | Description | Value | Default |
+| - | - | - | - |
+| **Enable** | Enables the exposure control properties. | Boolean | `Enabled`  |
+| **Overrides - Enabled Override** | If enabled, you can override Exposure Control settings to have advanced PostFX looks and control. | Boolean | `Enabled` |
```
[the remark this answers is not in the document]
Comment (chanmosq) [reply content/docs/user-guide/components/reference/atom/exposure-control.md:30]: Revised this part differently from your suggestion for more accuracy: 

>If enabled, you can override Exposure Control settings to have more control over how the PostFX behaves and looks.

**Review by FiniteStateGit — APPROVED** [review]

**Pull request merged**